app/src/main/java/grinnell/appdev/edu/lyles/AsyncRetrieval.java

```diff
@@ -0,0 +1,69 @@
+package grinnell.appdev.edu.lyles;
+
+import android.os.AsyncTask;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.IOException;
+
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+
+/**
+ * Created by Shelby on 2/3/2017.
+ */
+
+public class AsyncRetrieval extends AsyncTask<Void, Void, String> {
+
+    private final String url;
+    private String body;
```

Comment:
See comment above. I think we might want to be a bit more descriptive with the variable name here. :)